Sprint-1/implement/sum.js

```diff
@@ -1,4 +1,12 @@
 function sum(elements) {
+    const numbers = elements.filter(element => typeof element === 'number');
+    if (numbers.length === 0) {
+        return 0;
+    }
+    let totalSum =  numbers.reduce((accu ,curr)=> accu + curr,0)
+    return totalSum
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function return the value you expected?

```
sum([NaN, 1]);
sum([Infinity, -Infinity]);
```